flutter/lib/src/event_processor/native_app_start_event_processor.dart

```diff
@@ -1,35 +1,27 @@
 import 'dart:async';
-import 'package:meta/meta.dart';
 
 import 'package:sentry/sentry.dart';
 
-import '../sentry_native_state.dart';
-import '../sentry_native_wrapper.dart';
+import '../sentry_native.dart';
+import '../sentry_native_channel.dart';
 
 /// EventProcessor that enriches [SentryTransaction] objects with app start
 /// measurement.
-@internal
 class NativeAppStartEventProcessor extends EventProcessor {
   NativeAppStartEventProcessor(
-    this._nativeWrapper,
-    this._nativeState,
+    this._native,
   );
 
-  final SentryNativeWrapper _nativeWrapper;
-  final SentryNativeState _nativeState;
-
-  var _didFetchAppStart = false;
+  final SentryNative _native;
 
   @override
   FutureOr<SentryEvent?> apply(SentryEvent event, {hint}) async {
-    final appStartEnd = _nativeState.appStartEnd;
+    final appStartEnd = _native.appStartEnd;
```

Comment:
What if we don't use this EventProcessor anymore and do it within the `onFinish` as well? Does it make sense?

Reply:
Think this was one of the first feedback we had to move it away from the `SentryNavigatorObserve` so that users who don't init the observer would also have this feature.

Reply:
Indeed, makes total sense.